gtsfm/utils/geometry_comparisons.py

```diff
@@ -40,6 +43,122 @@ def align_rotations(aRi_list: List[Rot3], bRi_list: List[Rot3]) -> List[Rot3]:
     return [aRb.compose(bRi) for bRi in bRi_list]
 
 
+def ransac_align_poses_sim3_ignore_missing(
+    aTi_list_ref: List[Optional[Pose3]],
+    bTi_list_est: List[Optional[Pose3]],
+    num_iters: int = 1000,
+    delete_frac: float = DEFAULT_RANSAC_ALIGNMENT_DELETE_FRAC,
+    verbose: bool = False,
```

Comment:
I think we should use the powerful verbosity settings in the logger instead of creating more arguments.

Reply:
Good point. I'll update

Reply:
Updated.